cabal.project

```diff
@@ -22,6 +22,8 @@ packages: plutus-core
           prettyprinter-configurable
           doc
           web-ghc
+          plutus-common
+          plutus-errors
```

Comment:
I think `stack.yaml` also needs these to be added, but then that's kind of unofficial, so I'm not sure whose job it is to fix it...